apt/private/deb_translate_lock.bzl

```diff
@@ -107,6 +101,76 @@ filegroup(
 )
 """
 
+def _package_build(template, package, repo_name):
```

Comment:
Okay as part of this PR this doesn't has to change and it makes it 10x harder to understand a PR that could have been really simple to understand.  Can you revert this so i can actually see the diff?